src/pages/docs/deprecations/index.md

```diff
@@ -42,6 +42,15 @@ AzureRM was [deprecated by Microsoft](https://learn.microsoft.com/en-us/powershe
 AzureRm will remain available until July 2024 (with an in-app warning). After this, you'll need to move to either `az cli` or the `az module for powershell` 
 for Azure authentication.
 
+## Deprecations for 2024.2
+
+### Bundled Tools
+Command line tools for AWS, Azure and Terraform have been included with Octopus Deploy as a convenience mechanism for some time. The provided versions of these tools are now out of date and won't be updated.
+
+As of `2024.2`, deployment steps will no longer be able to use the bundled tools and they will be removed entirely in `2025.1`.
```

Comment:
```suggestion
As of `2024.2`, deployment steps will no longer be able to use the bundled tools, which will be removed in `2025.1`.
```

Reply:
This phase doesn't seem clear, if it's not removed until 2025.1, what does it mean to say it can't be used in 2024.2?

Reply:
Updated it to explicitly mention that they won't be able to be configured, that makes better sense, yes?